ERCS/erc-7682.md

```diff
@@ -86,9 +146,25 @@ An alternative we considered is defining a way for apps to fetch available auxil
 
 The shape of this capability allows for a more advanced extension if apps feel more functionality is needed.
 
+### Extensibility
+- The structure of `assetsRequiredForExecution` allows for future extension, such as specifying deadlines, slippage, or preferred bridges. Future versions of this standard MAY introduce additional fields as needed.
+
+### Decimals Source of Truth
+- Applications and wallets SHOULD verify the `decimals` value for each asset against onchain data or trusted sources to avoid errors from incorrect values.
```

Comment:
in this case, why have the dapp provide `decimals` at all? to make sure that the value they're trying to request via the `amount` field is what they intend?

Reply:
I've decided to remove the decimals property but add a section explaining that the wallet must cross reference the asset's decimals value to ensure it is interpreting values correctly:

https://github.com/ethereum/ERCs/pull/1126/commits/7fb4cf1d19c2e92a8bfbd10e8f948a858a9bca68

>The wallet MUST cross-check the `amount` value in `assetsRequiredForExecution` against the `decimal()` value specified on the `asset`'s contract to ensure that the amount is interpreted correctly according to the asset's decimal precision.